app/views/hiring_staff/vacancies/application_details/show.html.haml

```diff
@@ -1,9 +1,9 @@
 - content_for :page_title_prefix, page_title_prefix(@vacancy, @application_details_form, t('jobs.application_details'))
-= render 'hiring_staff/vacancies/vacancy_form_partials/title'
+= render 'hiring_staff/vacancies/vacancy_form_partials/heading'
 
 - if @vacancy.present? && @vacancy&.state != 'create'
   = link_to t('buttons.cancel_and_return'), edit_organisation_job_path(@vacancy.id), class: 'govuk-back-link'
-- else
+- elsif @vacancy.present?
```

Comment:
What's the reason for this change?

Reply:
I put a note in the commit message for `Prevent nil errors in views`: 
```
    I noticed that although in each case the first
    conditional checked for the presence of @vacancy, the else-case
    did not do likewise.
    This commit makes the two parts of the conditional consistent
    by having them both check for @vacancy.present? before using
    @vacancy.
```

Reply:
We could in theory get rid of @vacancy.present? in both parts of the conditional, as line 1 will throw a nil error in any case.

Reply:
> We could in theory get rid of @vacancy.present? in both parts of the conditional, as line 1 will throw a nil error in any case.

Tried this, didn't work.